src/lighthouse.php

```diff
@@ -59,6 +59,18 @@
 
     'guard' => 'api',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Guards
+    |--------------------------------------------------------------------------
+    |
+    | Specify here the guards that you have defined in your application and that are different from guard option.
+    | They will be used to check authentication in directives such as @can.
```

Comment:
I don't get it, this seems to just seems to just overrule `guard`?

Reply:
No, the idea was to indicate in the Lighthouse configuration the custom guards that are defined in the application to be able to check against them in the can directive, but you can also simply take those that are defined in Laravel (or in Lumen).